### PR TITLE
Add ref source to "view desktop" link

### PR DESCRIPTION
**Pull request by ajacksified**
:eyeglasses: @curioussavage or @danehansen

### CODE REVIEW

**Comment by curioussavage** [comment]
I am getting `?&ref_source=mweb`

**Comment by curioussavage** [comment]
:haircut:

**Comment by ajacksified** [comment]
:haircut:

File: src/views/components/SideNav.jsx [hunk 1]
```diff
@@ -44,7 +44,9 @@ class SideNav extends React.Component {
 
   _desktopSite(e) {
     e.preventDefault();
-    this.props.app.emit('route:desktop', this.props.url);
+    var url = this.props.url;
+    var query = querystring.stringify(this.props.query || {});
+    this.props.app.emit('route:desktop', `${url}?${query}`);
```
Comment (curioussavage) [inline src/views/components/SideNav.jsx]: could the `?` only be added if `query` isn't empty?

Comment (curioussavage) [inline src/views/components/SideNav.jsx]: :nail_care:

Comment (ajacksified) [inline src/views/components/SideNav.jsx]: argh, didn't push my changes.

**Comment by ajacksified** [comment]
:haircut:

**Comment by curioussavage** [comment]
:fish:

**Pull request merged**